### PR TITLE
Broker to push Resource Usage in JSON format periodically

### PR DESCRIPTION
**Pull request by kthare10**
Broker to push Resource Usage in JSON format periodically

https://github.com/fabric-testbed/ControlFramework/issues/78

### CODE REVIEW

File: fabric_cf/actor/core/core/broker_policy.py [hunk 1]
```diff
@@ -119,12 +121,16 @@ def get_client_id(self, *, reservation: ABCServerReservation) -> ID:
         return reservation.get_client_auth_token().get_guid()
 
     @staticmethod
-    def get_broker_query_model_query(*, level: int) -> dict:
+    def get_broker_query_model_query(*, level: int, bqm_format: int = GraphFormat.GRAPHML.value) -> dict:
```
Comment (ibaldin) [inline fabric_cf/actor/core/core/broker_policy.py:124]: Are you sure you need .value and can't just use GraphFormat.GRAPHML?

File: fabric_cf/actor/core/kernel/broker_query_model_publisher.py [hunk 2]
```diff
@@ -51,7 +52,7 @@ def execute(self):
         Process a claim timeout
         """
         try:
-            request = BrokerPolicy.get_broker_query_model_query(level=1)
+            request = BrokerPolicy.get_broker_query_model_query(level=1, bqm_format=GraphFormat.JSON_NODELINK.value)
```
Comment (ibaldin) [inline fabric_cf/actor/core/kernel/broker_query_model_publisher.py:55]: Same here - are you sure you need .value?

File: fabric_cf/actor/core/core/broker_policy.py (hunk 1 above)
Comment (kthare10) [reply fabric_cf/actor/core/core/broker_policy.py]: Updated the fix

**Review by ibaldin — APPROVED** [review]
All good

**Pull request merged**